site/content/3.11/develop/integrations/spring-data-arangodb/_index.md

```diff
@@ -184,22 +250,74 @@ public ArangoDB.Builder arango() {
   return arango;
 }
 ```
-
 {{< /tab >}}
 
 {{< /tabs >}}
 
+### Create a CommandLineRunner
+
+To run our demo as command line application, we have to create a class implementing `CommandLineRunner`:
+
+```java
+@ComponentScan("com.arangodb.spring.demo")
+public class CrudRunner implements CommandLineRunner {
+
+    @Autowired
+    private ArangoOperations operations;
+
+    @Autowired
+    private CharacterRepository repository;
+
+    @Override
+    public void run(String... args) {
+        // first drop the database so that we can run this multiple times with the same dataset
+        operations.dropDatabase();
+
+        System.out.println("# CRUD operations");
+
+        // save a single entity in the database
+        // there is no need of creating the collection first. This happen automatically
+        Character nedStark = new Character(null, "Ned", "Stark");
+        Character saved = repository.save(nedStark);
+        System.out.println("Ned Stark saved in the database: " + saved);
+    }
+}
+```
+
+### Run the application
+
+Finally, we create a main class:
+
+```java
+@SpringBootApplication
+public class DemoApplication {
+  public static void main(final String... args) {
+    System.exit(SpringApplication.exit(
+            SpringApplication.run(CrudRunner.class, args)
+    ));
+  }
+}
+```
+
+And run it with:
+
+```shell
+mvn spring-boot:run
+```
+
+This should produce a console output similar to:
+
+```
+Ned Stark saved in the database: Character[id=2029, name=Ned, surname=Stark]
+```
+
```

Comment:
Is this equally applicable to v3 and v4?

Reply:
Yes, it should be, but v3 EOL now.